### DISCUSSION TITLE
4.3.0

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#102] [Announcements]
Blender 4.3.0, commit [2b18cad88b13](https://projects.blender.org/blender/blender/commit/2b18cad88b13).
Build environment:

    Visual Studio 2019, version 16.11.42
    Windows SDK 10.0.22000
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

The experimental Vulkan backend is not functional on Windows 7 because there are no drivers that support VK_KHR_dynamic_rendering.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/4.3.0'>4.3.0</a>.</em>

**Comment by weihaojie** [comment]
blender4.5
